src/bp/ui-shared/src/translations/es.json

```diff
@@ -0,0 +1,256 @@
+{
+  "action": "Acción",
+  "actionRequired": "Acción requerida",
+  "add": "Añadir",
+  "addVariable": "Agregar variable",
+  "advancedSettings": "Ajustes avanzados",
+  "all": "Todos",
+  "and": "y",
+  "back": "Atrás",
+  "bug": "Bug",
+  "builder": "Constructor",
+  "cancel": "Cancelar",
+  "card": "Tarjeta",
+  "change": "Cambiar",
+  "channel": "Canal",
+  "chips": "Chips",
+  "chooseFile": "Elegir archivo...",
+  "close": "Cerrar",
+  "code": "Código",
+  "codeNewAction": "Código nueva acción",
+  "collapseAll": "Desplegar todo",
+  "converseApi": "Converse API",
+  "commander": {
+    "backToAdmin": "Volver al Admin",
+    "category": {
+      "admin": "Admin",
+      "command": "Comando",
+      "external": "Externo",
+      "module": "Módulo",
+      "studio": "Estudio"
+    },
+    "goTo": "Navegar a {destination}",
+    "inNewTab": "Abrir {destination} en una pestaña nueva",
+    "links": {
+      "chat": "chat",
+      "documentation": "documentación"
+    },
+    "searchCommand": "Buscar un comando",
+    "switchBot": "Cambiar a bot: {name}",
+    "switchWorkspace": "Espacio de trabajo: {nombre}",
+    "toDismiss": "to dismiss",
+    "toNavigate": "to navigate",
+    "toSelect": "to select",
+    "typeCommand": "Escribe un comando",
+    "viewBot": "Ver bot: {name}"
+  },
+  "confidence": "Confidencia",
+  "configuration": "Configuración",
+  "confirmation": "Confirmación",
+  "confirmPrompt": "Confirmación requerida",
+  "content": "Contenido",
+  "copy": "Copiar",
+  "count": "Cantidad",
+  "create": "Crear",
+  "createdOn": "Creado el",
+  "delete": "Eliminar",
+  "deleteCondition": "Eliminar condición",
+  "deleteContent": "Eliminar contenido",
+  "deleteImage": "Eliminar Imagen",
+  "deleteField": "Eliminar campo",
+  "deleteNode": "Eliminar nodo",
+  "deletePrompt": "Delete Prompt",
+  "deleteVariable": "Eliminar Variable",
+  "deleteWorkflow": "Eliminar flujo de trabajo",
+  "description": "Descripción",
+  "details": "Detalles",
+  "disabled": "deshabilitado",
+  "duplicate": "Duplicar",
+  "edit": "Editar",
+  "email": "E-mail",
+  "enable": "Habilitar",
+  "enabled": "Habilitado",
+  "error": "Error",
+  "execute": "Ejecutar",
+  "expandAll": "Expand AllExpandir todo",
+  "export": "Exportar",
+  "exportToJson": "Exportar a JSON",
+  "failure": "Fracaso",
+  "filter": "Filtro",
+  "filterBy": "Filtrado por",
+  "flows": "Flujos",
+  "general": "General",
+  "id": "ID",
+  "if": "If",
+  "ifElse": "If/Else",
+  "image": "Imagen",
+  "import": "Importar",
+  "importJson": "Importar JSON",
+  "insert": "Insertar",
+  "intent": "intent",
+  "isoLangs": {
+    "ar": {
+      "name": "Arabic",
```

Comment:
in those language fields, in the name attribute enter translated name.  Check my english suggestion and apply the rest to other languages please.